README.md

```diff
@@ -29,26 +29,56 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
 
 ### Template Parameters
 
+The following template parameters are used to configure CloudFormation resources
+
+| Template Parameter | Description |
+| SsmAliasPrefix | Prepend this value to the Alias Name of the KMS key created. |
+| SsmKeyAdminArn | Grant KMS admin access to these ARNs for managing secure secrets. |
```

Comment:
some of these are required parameters, would be good to add them to the example in `Install Lambda into AWS` section.

also this table does not look like a table in rendered html view.

Reply:
whoops, I'll fix the formatting